source/type/data.cpp

```diff
@@ -63,7 +63,7 @@ namespace pt = boost::posix_time;
 namespace navitia {
 namespace type {
 
-const unsigned int Data::data_version = 72;  //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 73;  //< *INCREMENT* every time serialized data are modified
```

Comment:
```suggestion
const unsigned int Data::data_version = 72;  //< *INCREMENT* every time serialized data are modified
```
no need to update `data_version` since serialization isn't changed